osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java

```diff
@@ -405,7 +423,7 @@ private List<AttributeAddress> getAttributeAddresses(final List<CosemObject> obj
 
   private CosemObject createCosemObject(final String tag, final int classId, final String obis) {
     return new CosemObject(
-        tag, "descr", classId, 0, obis, "group", null, List.of(), Map.of(), List.of());
+        tag, "descr", classId, 0, obis, "group", null, List.of(), Map.of(), new ArrayList<>());
```

Comment:
Why is `List.of()` changed to `new ArrayList<>()`?

Reply:
Omdat er verderop in de test waarden aan de list worden toegevoegd, bij een List mag dat niet meer